Spigot-Server-Patches/0161-Add-server-name-parameter.patch

```diff
@@ -0,0 +1,28 @@
+From db00d76af9d7de391aa9ccc59672ce542d2490d0 Mon Sep 17 00:00:00 2001
+From: Martin Panzer <postremus1996@googlemail.com>
+Date: Sat, 28 May 2016 12:01:49 +0200
+Subject: [PATCH] Add server-name parameter
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/Main.java b/src/main/java/org/bukkit/craftbukkit/Main.java
+index 979adad..1d8555f 100644
+--- a/src/main/java/org/bukkit/craftbukkit/Main.java
++++ b/src/main/java/org/bukkit/craftbukkit/Main.java
+@@ -130,6 +130,14 @@ public class Main {
+                         .defaultsTo(new File("paper.yml"))
+                         .describedAs("Yml file");
+                 // Paper end
++
++                // Paper Start
```

Comment:
The already existing multi line comments in this file also use `// Paper Start` and end with `// Paper end`. I use the same style, so comments in this file are consistent. Should I change it anyhow?